### PR TITLE
Update release instructions + release v3.0

### PR DESCRIPTION
**Pull request by jamietanna**
It probably makes sense for me to get this merged, then get it released, so it's the exact version from `master` at the time of upload :+1:

### CODE REVIEW

**Comment by snarfed** [comment]
woo, exciting!

one other thing i do at this point that's not in the release instructions is open the generated html docs in the browser and skim them to make sure they're reasonable and nothing big is broken or missing. looks like you ran `./docs/build.sh`, so you should be able to open `docs/_build/html/index.html`. mind trying that? (we should probably also add this to the release steps.)

**Comment by jamietanna** [comment]
@snarfed should be ready for re-review!

File: README.md [hunk 1]
```diff
@@ -296,7 +299,7 @@ On the open source side, there are many related projects. [php-mf2-shim](https:/
 
 Changelog
 ---
-### 3.0 - unreleased
+### 3.0 - 2020-03-29
```
Comment (snarfed) [inline README.md:302]: mind updating this to the date you think you'll be ready to finish the release and push it to pypi? can be today, or in the future. (and then run `./docs/build.sh` again.)

**Comment by jamietanna** [comment]
@snarfed that's ready to go - but when trying to install from test.pypi.org I receive:

```sh
$ pip install -i https://test.pypi.org/simple --extra-index-url https://pypi.org/simple granary==$ver
Looking in indexes: https://test.pypi.org/simple, https://pypi.org/simple
Collecting granary==3.0
  Downloading https://test-files.pythonhosted.org/packages/a7/95/d02a526514df8c3e66efed0a2202504ae4718341f3a511d1ac3326168f02/granary-3.0.tar.gz (344kB)
    100% |################################| 348kB 2.4MB/s 
Collecting beautifulsoup4~=4.8 (from granary==3.0)
  Downloading https://test-files.pythonhosted.org/packages/6d/7a/342cca66a97eb7727886e20b54e6d531d538fc1b5cfbfe61ac08a40f6c76/beautifulsoup4-4.9.0-py2-none-any.whl (109kB)
    100% |################################| 112kB 5.4MB/s 
Collecting brevity>=0.2.17 (from granary==3.0)
Could not install packages due to an EnvironmentError: 404 Client Error: Not Found for url: https://test.pypi.org/simple/brevity/
```

**Comment by jamietanna** [comment]
(I've bumped the date to tomorrow, given I'm not going to be able to get this live today as it's getting late here in the UK - hopefully we can get it out the door tomorrow!)

**Comment by snarfed** [comment]
> @snarfed that's ready to go - but when trying to install from test.pypi.org I receive:

great! and huh, it's saying that the brevity package isn't on test.pypi.org, but `--extra-index-url https://pypi.org/simple` should make it fall back to real pypi. are you using a recent verson of pip? it works for me with pip 20.0.2.

no matter though, you should be able to work around it with `pip install brevity` first and then the `pip install -i ...` from the instructions.

**Comment by jamietanna** [comment]
@snarfed looks like it's ready to release - I guess you'd need to merge then tag + draft the GH release?

**Comment by snarfed** [comment]
yes! i'm actually going to add you as a committer on this repo, and then let you follow the rest of the instructions. no good deed goes unpunished! 😁

**Comment by snarfed** [comment]
feel free to merge and continue on whenever you're ready. i'll also be in #indieweb-dev!

**Comment by snarfed** [comment]
also i've added readthedocs instructions to the end of the release process in https://github.com/snarfed/granary/pull/196 . low priority, but we can maybe try them after the rest is done. you'll first need to create a user on https://readthedocs.org/, and then i'll add you.

sorry for all the hoops to jump through! you're already through most of the hard parts!

**Comment by jamietanna** [comment]
@snarfed looks like I can't yet merge it:

>  Only those with write access to this repository can merge pull requests. 

Mind adding my access?

**Comment by snarfed** [comment]
i invited you! check your email? i think you need to accept first.
image

**Comment by jamietanna** [comment]
Ah gotcha - accepted now, so I'll push this forward :+1:

**Pull request merged**